Add marketing campaign landing page for services sign up

CroftLA opened the pull request:
Add the 'Helping you become a teacher' campaign landing page

![helping-you-become-a-teacher-campaign](https://user-images.githubusercontent.com/128088/102369139-8485f700-3fb3-11eb-9276-cdf135f34caf.png)

CroftLA committed:
Create helping-you-become-a-teacher

CroftLA committed:
add new campaigns page content

peteryates commented:
@CroftLA - looks good so far, once DFE-Digital/get-into-teaching-app/pull/647 is merged we can start adding the CTAs in 👍🏽

CroftLA committed:
Created the sections for the helping you become a teacher page

peteryates committed:
Convert helping to become a teacher page to erb

This will allow us to render calls to action between sections

peteryates committed:
Add a few sample CTAs with placeholder text

peteryates committed:
Add placeholders for remaining CTAs

peteryates committed:
Amend the CTA copy and reorganise the document

peteryates committed:
Update the helping to become a teacher copy

peteryates committed:
Reorganise page so it's defined in frontmatter

Front matter is required by the app, declaring it as an instance
variable in the top of the view means that the page can't be added to
navigation and it causes other rendering and display issues.

Now the list of partials that make up the page are listed under
`content:` and rendered sequentially

peteryates committed:
Add an intro paragraph to the campaign page

MylesJarvis reviewed (commented):
Really impressed with this, great work both. 

We've chatted but for reference:
Comments/amendments I'll need are:
<img width="656" alt="Screenshot 2020-12-16 at 15 56 50" src="https://user-images.githubusercontent.com/35870975/102373120-c618a100-3fb7-11eb-8231-9f51f624239e.png">

Can we split out the Email List sentence onto a new line.

<img width="688" alt="Screenshot 2020-12-16 at 15 58 46" src="https://user-images.githubusercontent.com/35870975/102374648-7f2bab00-3fb9-11eb-9be9-7b42dcbf48ae.png">


Looks like there's a slight duplication/style point here on descriptions of Events, slight tweak needed to prevent feeling of repetition.

<img width="678" alt="Screenshot 2020-12-16 at 15 57 13" src="https://user-images.githubusercontent.com/35870975/102374431-3ecc2d00-3fb9-11eb-92a6-632d81336055.png">

Final line on recent trainees benefitting needs to be removed.

peteryates committed:
Some minor rewording based on feedback

peteryates committed:
Further rewording of the events campaign section

peteryates committed:
Add a missing word to email section

MylesJarvis reviewed (approved):
Approved - let's deploy tomorrow AM.
